java/code/src/com/redhat/rhn/common/security/acl/Access.java

```diff
@@ -499,4 +500,18 @@ public boolean aclErrataEditable(Object ctx, String[] params) {
         Errata e = ErrataFactory.lookupById(eid);
         return e != null && e.getOrg() != null;
     }
+
+    public boolean aclFormulaValueEquals(Object ctx, String[] params) {
```

Comment:
Is this used somewhere? I cannot find any reference to it.

Reply:
Yes, here https://github.com/uyuni-project/uyuni/pull/1661/files#diff-85595dcbc480319b138af97b9d2e669aR53